proxysql/qa-proxysql2-pipeline.groovy

```diff
@@ -23,32 +23,12 @@ pipeline {
             name: 'PROXYSQL_PACKAGE_BRANCH',
             trim: true)
         string(
-            defaultValue: 'https://github.com/percona/percona-xtradb-cluster',
-            description: 'URL to PXC repository',
-            name: 'PXC57_REPO',
-            trim: true)
-        string(
-            defaultValue: '5.7',
-            description: 'Tag/Branch for PXC repository',
-            name: 'PXC57_BRANCH',
-            trim: true)
-        string(
-            defaultValue: 'https://github.com/percona/percona-xtrabackup',
-            description: 'URL to PXB24 repository',
-            name: 'PXB24_REPO',
-            trim: true)
-        string(
-            defaultValue: 'percona-xtrabackup-2.4.20',
-            description: 'Tag/Branch for PXC repository',
-            name: 'PXB24_BRANCH',
-            trim: true)
-        string(
-            defaultValue: 'v2.0.13-dev',
+            defaultValue: 'v2.0.15-dev',
             description: 'Tag/Branch for ProxySQL-admin-tool repository',
             name: 'PAT_TAG',
             trim: true)
        choice(
-            choices: 'centos:6\ncentos:7\ncentos:8\nubuntu:xenial\nubuntu:bionic\nubuntu:focal\ndebian:stretch\ndebian:buster',
```

Comment:
Why CentOS 8 was removed?

Reply:
The proxysql-packaging build script is not working on CentOS 8. Will add CentOS:8 after fixing the issue.
https://pxc.cd.percona.com/job/qa-proxysql2-pipeline/206/console